recipes-app/node-red-gpio/files/0007-mraa-gpio-Fix-led-label-in-the-flow-editor.patch

```diff
@@ -0,0 +1,37 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Baocheng Su <baocheng.su@siemens.com>
+Date: Thu, 28 Dec 2023 11:00:06 +0800
+Subject: [PATCH] mraa-gpio: Fix led label in the flow editor
+
+The default label for led node is 'led', which is almost useless.
+Change it to pattern "USER<n> LED <color>".
```

Comment:
BTW, this name scheme is only valid because the IOT2050 is the only user of the led module.

Reply:
Added into commit message